eng/build.ps1

```diff
@@ -61,7 +61,7 @@ if ($help -or (($null -ne $properties) -and ($properties.Contains('/help') -or $
 
 # VS Test Explorer support for libraries
 if ($vs) {
-  Write-Host "!!! VS Test Explorer now works without the -vs switch. The switch will be removed eventually. !!! "
+  Write-Host "VS Test Explorer now works without needing to call build.cmd. The -vs switch will be removed eventually."
```

Comment:
FWIW this also removes the !!! characters, in case you wanted to still keep them

Reply:
Yeah that was intentional